src/utils/string/StringUtils.js

```diff
@@ -11,9 +12,10 @@ import TextBold from '../../components/Text/TextBold';
 import TopicText from '../../components/TopicText';
 // eslint-disable-next-line import/no-cycle
 import removePrefixTopic from '../topics/removePrefixTopic';
+import {COLORS} from '../theme';
+import {DEFAULT_PROFILE_PIC_PATH, DEFAULT_TOPIC_PIC_PATH} from '../constants';
 import {getAnonymousUserId} from '../users';
 import {getUserId} from '../token';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The order of import statements has been changed. While this doesn't affect the functionality of the code, it's generally a good practice to maintain a consistent order of imports for readability and maintainability. Typically, absolute imports are placed before relative imports, and within these categories, they are sorted alphabetically.
```diff
- import {COLORS} from '../theme';
- import {DEFAULT_PROFILE_PIC_PATH, DEFAULT_TOPIC_PIC_PATH} from '../constants';
- import {getAnonymousUserId} from '../users';
- import {getUserId} from '../token';
+ import {getAnonymousUserId} from '../users';
+ import {getUserId} from '../token';
+ import {COLORS} from '../theme';
+ import {DEFAULT_PROFILE_PIC_PATH, DEFAULT_TOPIC_PIC_PATH} from '../constants';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->